packages/local_auth/ios/Classes/LocalAuthPlugin.m

```diff
@@ -62,16 +68,26 @@ - (void)alertMessage:(NSString *)message
                                                                                    completion:nil];
 }
 
+- (BOOL)isFingerprintAvailable{
+  LAContext *context = [[LAContext alloc] init];
+  NSError *authError = nil;
+  BOOL isAvailable = false;
+  if([context canEvaluatePolicy:LAPolicyDeviceOwnerAuthenticationWithBiometrics
+              error: &authError]){
+      isAvailable = true;
+  }
+  return isAvailable;
+}
+
 - (void)authenticateWithBiometrics:(NSDictionary *)arguments
                  withFlutterResult:(FlutterResult)result {
   LAContext *context = [[LAContext alloc] init];
   NSError *authError = nil;
   lastCallArgs = nil;
   lastResult = nil;
   context.localizedFallbackTitle = @"";
-
-  if ([context canEvaluatePolicy:LAPolicyDeviceOwnerAuthenticationWithBiometrics
-                           error:&authError]) {
+  BOOL isAvailable = [self isFingerprintAvailable];
```

Comment:
I don't think we need to actually create a local variable for this just to check it in the conditional.